draft-ietf-rtcweb-jsep.xml

```diff
@@ -1433,18 +1479,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <t>The next step is to generate m= sections, as specified in
           <xref target="RFC4566"/> Section 5.14, for each
-          MediaStreamTrack that has been added to the PeerConnection via the
-          addStream method. (Note that this method takes a MediaStream, which
-          can contain multiple MediaStreamTracks, and therefore multiple m=
-          sections can be generated even if addStream is only called once.)
-          m=sections MUST be sorted first by the order in which the
-          MediaStreams were added to the PeerConnection, and
-          then by the alphabetical ordering of the media type for the MediaStreamTrack.
-          For example, if a MediaStream containing both an audio and a video
-          MediaStreamTrack is added to a PeerConnection, the resultant m=audio
-          section will precede the m=video section. If a second MediaStream
-          containing an audio MediaStreamTrack was added, it would follow
-          the m=video section.</t>
+          RtpTransceiver that has been added to the PeerConnection via the
+          addTrack or addTransceiver methods. m=sections MUST be sorted
```

Comment:
Agree with the policy, but think we could word this better, e.g. "m= sections are generated in the order that their associated transceivers were added to the PeerConnection."